### ISSUE TITLE
Add own options to custom toast component

### ISSUE THREAD

**Issue by frct1** [scttcper/ngx-toastr#766]
Hi,
Today found one use case for passing custom options. In my case i want to pass icon name to component. It is actually possible with directly passing but there is errors about
`Object literal may only specify known properties, and 'toastIcon' does not exist in type 'Partial<IndividualConfig>'`

Is there any workarounds for this ?

**Comment by scttcper** [comment]
```ts
const inserted = this.toastr.show('message', 'title');
inserted.toastRef.componentInstance.myProperty = 'something';
```
can access the component instance directly after creation

**Comment by frct1** [comment]
I've now looked at componentInstance, showing that it is undefined. Strange

![image](https://user-images.githubusercontent.com/25178244/72224855-fa0d6a00-358f-11ea-9e9e-cd75d9dd9f75.png)

**Comment by danhooper** [comment]
componentInstance is undefined for me as well when I upgrade to Angular 9.

**Comment by scttcper** [comment]
Will be fixed in #778

**Comment by frct1** [comment]
@scttcper 
Works pretty good. Thank you very much :)
